localisation/00_wc_job_titles.csv

```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
 job_spiritual_female_void_group;Shadow Priestess;Pr�tresse des Ombres;Schattenj�gerin;;;;;;;;;;;x
 job_spiritual_vrykul_religion_group;High Sage;Grand sage;Hochweiser;;;;;;;;;;;x
 job_spiritual_female_vrykul_religion_group;High Sage;Grand sage;Hochweise;;;;;;;;;;;x
+#Cultural Titles;;;;;;;;;;;;;;x
+job_chancellor_dalaranian;Second Counselor;Second Conseiller;Zweites Ratsmitglied;;;;;;;;;;;x
+job_chancellor_female_dalaranian;Second Counselor;Seconde Conseill�re;Zweites Ratsmitglied;;;;;;;;;;;x
+job_marshal_dalaranian;Third Counselor;Troisi�me Conseiller;Drittes Ratsmitglied;;;;;;;;;;;x
+job_marshal_female_dalaranian;Third Counselor;Troisi�me Conseill�re;Drittes Ratsmitglied;;;;;;;;;;;x
+job_treasurer_dalaranian;Fourth Counselor;Quatri�me Conseiller;Viertes Ratsmitglied;;;;;;;;;;;x
+job_treasurer_female_dalaranian;Fourth Counselor;Quatri�me Conseill�re;Viertes Ratsmitglied;;;;;;;;;;;x
+job_spymaster_dalaranian;Fifth Counselor;Cinqui�me Conseiller;F�nftes Ratsmitglied;;;;;;;;;;;x
+job_spymaster_female_dalaranian;Fifth Counselor;Cinqui�me Conseill�re;F�nftes Ratsmitglied;;;;;;;;;;;x
+job_chancellor_ancient_group;Ancient of Statecraft;Ancien d'�tat;Urtum der Staatskunst;;;;;;;;;;;x
+job_chancellor_female_ancient_group;Ancient of Statecraft;Ancienne d'�tat;Urtum der Staatskunst;;;;;;;;;;;x
+job_marshal_ancient_group;Ancient of War;Ancien de Guerre;Urtum des Krieges;;;;;;;;;;;x
+job_marshal_female_ancient_group;Ancient of War;Ancienne de Guerre;Urtum des Krieges;;;;;;;;;;;x
+job_treasurer_ancient_group;Ancient of Custody;Ancien de Garde;Urtum der Verwahrung;;;;;;;;;;;x
+job_treasurer_female_ancient_group;Ancient of Custody;Ancienne de Garde;Urtum der Verwahrung;;;;;;;;;;;x
+job_spymaster_ancient_group;Ancient of Shadows;Ancien des Ombres;Urtum der Schatten;;;;;;;;;;;x
+job_spymaster_female_ancient_group;Ancient of Shadows;Ancienne des Ombres;Urtum der Schatten;;;;;;;;;;;x
+job_chancellor_aqir_group;Vizier;Vizir;Wesir;;;;;;;;;;;x
+job_chancellor_female_aqir_group;Vizier;Vizir;Wesirin;;;;;;;;;;;x
+job_marshal_aqir_group;Blade Lord;Ma�tre des Lames;Meister der Klingen;;;;;;;;;;;x
+job_marshal_female_aqir_group;Blade Lady;Ma�tre des Lames;Meisterin der Klingen;;;;;;;;;;;x
+job_treasurer_aqir_group;Swarmspeaker;;Herold;;;;;;;;;;;x
+job_treasurer_female_aqir_group;Swarmspeaker;;Heroldin;;;;;;;;;;;x
+job_spymaster_aqir_group;Emissary;Emissaire;Gesandter;;;;;;;;;;;x
+job_spymaster_female_aqir_group;Emissary;Emissaire;Gesandte;;;;;;;;;;;x
```

Comment:
Emissary title is used by Chancellors. I suggest **_Saboteur_**.
https://wow.gamepedia.com/Saboteur_Kip%27tilak